packager/signer/signer.go

```diff
@@ -452,6 +452,13 @@ func formatLinkHeader(preloads []*rpb.Metadata_Preload) (string, error) {
 		value.WriteString(u.String())
 		value.WriteString(">;rel=preload;as=")
 		value.WriteString(preload.As)
+		for _, attr := range preload.GetAttributes() {
+			value.WriteByte(';')
+			value.WriteString(attr.Key)
+			value.WriteString(`="`)
+			value.WriteString(strings.ReplaceAll(attr.Val, `"`, `\"`))
```

Comment:
Use util.QuotedString instead: https://github.com/ampproject/amppackager/blob/ca6f35a1b2e65f02f8e55c9e9bc218f25b801dc8/packager/util/http.go#L102

(And I guess just skip the whole href if any of its param values are unquotable, for defense-in-depth against bugs in preloadimage.go.)

Reply:
Done.